### DISCUSSION TITLE
network connection problems

### DISCUSSION THREAD

**Discussion by tec81** [rustdesk/rustdesk#5930] [General]
### Bug Description

Getting fail messages on the bottom of the RustDesk window every second that the server isn't ready and I have to check network connection. Cannot get a stable connection.

### How to Reproduce

.

### Expected Behavior

.

### Operating system(s) on local side and remote side

Mac OS 12.7

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

.

**Comment by illatus** [comment]
I have the same issues days now. I can't figure out a solution.

**Reply by tec81** [reply]
I tried now to trigger 10 times to connect and sometimes it worked, sometimes not.
I don't know if I should try to install the system service again. the last four times there is nothing working.
